ballsdex/packages/players/cog.py

```diff
@@ -90,3 +91,26 @@ async def donation_policy(
             await interaction.response.send_message("Invalid input!")
             return
         await player.save()  # do not save if the input is invalid
+
+    @app_commands.command()
+    async def developer(self, interaction: discord.Interaction, player: discord.User | None):
+        """
+        Check if a player is considered as a developer.
+
+        Parameters
+        ----------
+        player: discord.User | None
+            The player you want to inspect, if not yourself.
+        """
+        user = player or interaction.user
+
+        try:
+            object = await PlayerModel.get(discord_id=user.id)
+        except DoesNotExist:
+            await interaction.response.send_message("The provided user is not a player.", ephemeral=True)
+            return
+        
+        if object.developer is True:
+            await interaction.response.send_message("Oh yeah that guy is a dev")
```

Comment:
I think this word fits better
```suggestion
            await interaction.response.send_message("Oh yeah that guy is a developer")
```